### PR TITLE
Memory leak fix

### PR DESCRIPTION
**Pull request by e-strokov**
Fix for the memory leaking in mesh instances.
MI references were not removed from materials.
VB and IB now are destroyed with MI.

### CODE REVIEW

File: src/plugin/spine.js [hunk 1]
```diff
@@ -254,6 +254,21 @@ pc.extend(pc, function () {
...
         this.removeFromLayers();
         this._meshes = [];
+        for (var i = 0, len = this._meshInstances.length; i < len; i++) {
+            mesh = this._meshInstances[i].mesh;
+            if (mesh) {
+                if (mesh.vertexBuffer) {
+                    mesh.vertexBuffer.destroy();
+                    mesh.vertexBuffer = null;
+                }
+                for (j = 0; j < mesh.indexBuffer.length; j++) {
```
Comment (willeastcott) [inline src/plugin/spine.js]: j not defined anywhere

File: src/plugin/spine.js [hunk 2]
```diff
@@ -254,6 +254,21 @@ pc.extend(pc, function () {
     Spine.prototype.rebuildMeshes = function () {
         this.removeFromLayers();
         this._meshes = [];
+        for (var i = 0, len = this._meshInstances.length; i < len; i++) {
+            mesh = this._meshInstances[i].mesh;
```
Comment (willeastcott) [inline src/plugin/spine.js]: Missing var?

File: src/plugin/spine.js (hunk 1 above)
Comment (e-strokov) [reply src/plugin/spine.js]: Fixed.

File: src/plugin/spine.js (hunk 2 above)
Comment (e-strokov) [reply src/plugin/spine.js]: Fixed

**Review by willeastcott — APPROVED** [review]

**Pull request merged**